harmony_changelog.md

```diff
@@ -0,0 +1,103 @@
+# MuditaOS changelog - Harmony
+
+## [1.6.0 2022-06-14]
+
+### Added
+
+#### Home Screen:
+* Possibility to choose Clock Face from 8 options (3 for 24h format, 5 for 12h format).
+
+#### Meditation:
+* Possibility to set a Meditation Timer start delay.
+
+#### UI/UX:
+* UI update (Home Screen settings).
+
+#### Translations:
+* Completed missing translations.
+
+### Fixed
+* Fixed problem with confirmation of language change action.
+* Fixed problem with mixed translations appearing after language change.
+* Fixed missing texts.
+* Fixed issue with the device restarting when trying to turn it off.
+* Fixed incorrect unwanted shutdown of device in transport mode.
+* Fixed problem with duplicated action after pressing back and lightclick button.
+* Fixed problem with blank screen when approving certain settings.
+* Fixed problem with Home Screen flashing during startup.
+* Fixed problem with relaxation sounds turning off after alarm or bed time deactivation.
+
+## [1.5.1 2022-04-07]
+
+### Fixed
+* Fixed issue with the device appearing in service mode when connected via USB.
+* Fixed problems with clock desynchronization.
+
+## [1.5.0 2022-02-25]
+
+### Added
+
+#### Settings:
+* Ability to scroll between Harmony settings.
+* The order of alarm sounds has been defined.
+
+### Changed / Improved
+* The default alarm volume has been changed.
+* Updated alarm sounds.
+* The welcome screen has been updated.
+
+### Fixed
+* Fixed problem with Pre wake-up light.
+* Fixed problems with turning on the device despite high battery level.
+* Fixed problems with incorrect display of battery charge status.
+* Fixed issue with the turning on the device when the battery is critical, even if a charger is plugged.
+* Fixed problem with the occurrence of duplicate texts.
+* Fixed problem with alarm sounds appearing after changing device language.
+* Fixed problem with opening the Menu section when confirming the alarm setting.
+* Fixed problem with resetting some settings after changing language.
+* Fixed gaps in translations.
+* Fixed a problem with setting the new alarm sound.
+
+## [1.4.0 2022-02-01]
+
+### Added
+#### General:
+* Transportation mode.
+
+#### PowerNap:
+* Add front light functionality to PowerNap app.
+
+### Changed/Improved
+#### UI/UX:
+
+* Changed exit screen with instruction to turn on.
+* Update Mudita logo icon.
+* Updated fonts.
+* Updated the screen at the end of the Onboarding process.
+
+#### Meditation:
+
+* Updated texts.
+* The time of appearance of messages has been extended.
```

Comment:
More compact version without two "of" - "Messages display time has been extended."